entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityOperationHandler.kt

```diff
@@ -38,8 +42,8 @@ class EntityOperationHandler(
             .map {
                     entityService.processBatchOfEntities(it.first, it.second, it.third)
             }
-            .flatMap {
-                ok().body(BodyInserters.fromValue(it))
+            .map {
+                ResponseEntity.ok().body(serializeObject(it))

```

Comment:
same here for `serializeObject`, I don't get why it is used now, whereas it was not needed before?